omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -1,15 +1,13 @@
-TRIAL: OMERO.web WSGI deployment
-================================
+OMERO.web deployment
```

Comment:
Sorry but can we restore the 'WSGI' in the title here to distinguish this from the deployment subsections elsewhere?